web/src/main/java/com/bazaarvoice/emodb/web/auth/DefaultRoles.java

```diff
@@ -126,11 +133,11 @@
 
     // Reserved role for anonymous access
     anonymous (
-            // TODO:  Lock this down.  For now this will permit all standard client operations.
+            // TODO:  Lock this down.  For now this will permit all standard client operations with excluding pii tables/placements in SOR.
             //        Additionally, Shovel historically did not require permission to create facades, only to
             //        update facade records.  To maintain backwards compatibility anonymous also has this permission.
-            ImmutableSet.of(standard),
-            Permissions.createFacade(ALL));
+            ImmutableSet.of(sor_standard_without_pii, blob_standard, queue_standard, databus_standard),
+            Permissions.createFacade(NON_SYSTEM_NON_PII_TABLE));
```

Comment:
As noted in the comment, shovel is the only service creating facades, and it now uses API keys which have facade permissions, so it may be cleaner to just remove `Permissions.createFacade()` entirely from anonymous.